README.md

```diff
@@ -649,9 +651,9 @@ await index.fetch(['1']);
 
 See [Use namespaces](https://docs.pinecone.io/guides/indexes/use-namespaces) for more information.
 
-### Upsert records
+### Upsert vectors
 
-Pinecone expects records inserted into indexes to have the following form:
+Pinecone expects vectors inserted into indexes to have the following form:
```

Comment:
Records is the correct terminology for vector values + id + metadata which is what you're dealing with here. https://docs.pinecone.io/guides/get-started/key-concepts#record

Reply:
ah okay, I'll change that one back then, thanks!